win32ss/gdi/ntgdi/freetype.c

```diff
@@ -2452,24 +2383,27 @@ FontFamilyInclude(LPLOGFONTW LogFont, PUNICODE_STRING FaceName,
 
 static BOOL FASTCALL
 FontFamilyFound(PFONTFAMILYINFO InfoEntry,
-                PFONTFAMILYINFO Info, DWORD InfoCount)
+                PFONTFAMILYINFO Info, DWORD InfoCount, WCHAR *pszSubstName)
```

Comment:
PCWSTR is preferable because it contains a `_Null_terminated_` annotation

Reply:
OK, done.